config/initializers/edition_services.rb

```diff
@@ -1,13 +1,17 @@
 Whitehall::Application.config.to_prepare do
   Whitehall.edition_services.tap do |coordinator|
     coordinator.subscribe do |event, edition, _options|
-      ServiceListeners::AttachmentUpdater.call(attachable: edition) unless %w[publish force_publish].include?(event)
+      ServiceListeners::AttachmentUpdater.call(attachable: edition) unless %w[publish force_publish delete].include?(event)
```

Comment:
Related to my earlier comment on this file - I can see now why you may want to split these out into separate `.subscribe` calls. In which case... what other events remain, apart from force_publish/publish/delete? It might be time to change the `.subscribe` call to only subscribe to the events we actually care about now.